src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSubscriptionEditModel.ts

```diff
@@ -160,7 +157,8 @@ class ongoingTaskSubscriptionEditModel extends ongoingTaskEditModel {
                 ResponsibleNode: null,
                 LastClientConnectionTime: null,
                 LastBatchAckTime: null,
-                MentorNode: null
+                MentorNode: null,
+                NodeTag: null
```

Comment:
What's the difference between mentor node and node tag? Do we need it during edit?

Reply:
this is part of the subscription model